### PR TITLE
feat: ui review changes

### PR DESCRIPTION
**Pull request by KenLSM**
## Problem

Closes FRM-1109

Reviews from design

Admin individual response page
- Update copy for header titles
- Add tooltip to mention payout dates
- Fix badge hover by ignoring pointer events

Admin response table
- update table width, ordering, and copy for response table

Variable payment preview
- Update copy for variable payment preview
- Fix input hint showing between 0 and 0, when paymentMax is not populated, on admin create and build page
- Update input hint to switch to error when payee input invalid values

Product modal Changes
- Updating of error copies
- Addition of new error message to better direct admin to fix `amount` field when `amount max qty` exceeds `maxPaymentAmountCents`
- Fix issue where errors doesn't appear when admins didn't input min / max amount

## Solution

**Breaking Changes** 

- [ ] No - this PR is backwards compatible  

## Before & After Screenshots

**BEFORE**:

**AFTER**:

| Changes | Before | After |
|--------|--------|--------|
| Admin individual response page | Screenshot 2023-08-21 at 4 33 16 PM | Screenshot 2023-08-21 at 4 33 29 PM |
| Admin response table |  Screenshot 2023-08-21 at 4 30 26 PM | Screenshot 2023-08-21 at 4 29 32 PM |
| Variable payment preview | Screenshot 2023-08-21 at 4 34 55 PM | Screenshot 2023-08-21 at 4 34 58 PM |
| Cell | Cell | Cell | 
| Product Modal - Qty x Amt Error prompt | Screenshot 2023-08-21 at 4 18 49 PM |  Screenshot 2023-08-21 at 4 19 03 PM | 
| Product Modal - Empty Qty Prompt |  Screenshot 2023-08-21 at 4 19 32 PM | Screenshot 2023-08-21 at 4 19 43 PM   | 

## Tests

**Admin individual response page**
- [ ] Observe that mouse-over on badge doesn't trigger any on-hover visual changes
- [ ] Observe that mouse-over tooltip will show payout information **above** the tooltip

**Admin response table**
On payment forms
- [ ] Observe that only headers have currency symbols

On non-payment forms
- [ ] Observe that payment headers are not rendered

**Variable payment preview**
Create and build page
- [ ] Observe that input hint is shown when `paymentMax` is not populated

### CODE REVIEW

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/ProductModal.tsx [hunk 1]
```diff
@@ -257,9 +267,12 @@ export const ProductModal = ({
                       render={({ field }) => (
                         <Input
                           {...register(MIN_QTY_KEY, {
-                            required: watchMultiQtyEnabled,
+                            required:
+                              watchMultiQtyEnabled &&
+                              'The minimum quantity is 1',
```
Comment (LinHuiqing) [inline frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/ProductModal.tsx:272]: Won't the string part always evaluate to true? Why is this added to the `required` conditional?

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/ProductModal.tsx [hunk 2]
```diff
@@ -92,20 +92,25 @@ export const ProductModal = ({
...
+          Number(centsToDollars(minPaymentAmountCents)),
+        )} and S${formatCurrency(
+          Number(centsToDollars(maxPaymentAmountCents)),
+        )}`
 
       const validateMin = !!val && dollarsToCents(val) >= minPaymentAmountCents
       // Repeat the check on minPaymentAmountCents for correct typing
       if (!!minPaymentAmountCents && !validateMin) {
-        return `The maximum amount is ${formatCurrency(
+        return `The maximum amount is S${formatCurrency(
```
Comment (LinHuiqing) [inline frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/ProductModal.tsx]: Typo that wasn't caught in previous design reviews / QAs? 👀 

```suggestion
        return `The minimum amount is S${formatCurrency(
```

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/ProductModal.tsx (hunk 1 above)
Comment (LinHuiqing) [reply frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/ProductModal.tsx:272]: Okay I get what it's doing now, pretty neat!

**Review by LinHuiqing — APPROVED** [review]
Small nit for typo, lgtm and tested otherwise!

**Comment by KenLSM** [comment]
Thanks @LinHuiqing for helping to ship this!
